go/kbfs/test/benchmark_test.go

```diff
@@ -231,72 +239,106 @@ func BenchmarkWriteMixedFilesNormalBandwidth(b *testing.B) {
 	}
 	b.SetBytes(totalSize)
 
-	benchmark(b, silentBenchmark{b},
+	benchmark(b,
 		users("alice"),
 		blockSize(512<<10),
 		bandwidth(11*1024/8 /* 11 Mbps */),
 		opTimeout(19*time.Second),
 		as(alice,
 			custom(func(cb func(fileOp) error) error {
-				b.ResetTimer()
-				defer b.StopTimer()
+				var n int
+				cb(getBenchN(&n))
+				cb(resetTimer())
+				defer cb(stopTimer())
 				currIter := 0
 				for _, fileSize := range fileSizes {
 					numWrites := int(fileSize / perWriteBytes)
 					if err := benchmarkDoBenchWrites(b, cb,
 						numWrites, buf, currIter); err != nil {
 						return err
 					}
-					currIter += b.N
+					currIter += n
 				}
 				return nil
 			}),
 		),
 	)
 }
 
-func BenchmarkMultiFileSync(b *testing.B) {
-	numFiles := 20
-	fileSize := 5
-	benchmark(b, silentBenchmark{b},
+func benchmarkMultiFileSync(
+	b *testing.B, numFiles, fileSize int, timeWrites, timeFlush bool) {
+	benchmark(b,
 		journal(),
 		users("alice"),
+		batchSize(20),
+		as(alice,
+			mkdir("a"),
+		),
 		as(alice,
 			enableJournal(),
+			pauseJournal(),
 			custom(func(cb func(fileOp) error) error {
-				files := make([]string, numFiles*b.N)
-				bufs := make([][]byte, numFiles*b.N)
-				for i := 0; i < numFiles*b.N; i++ {
-					files[i] = fmt.Sprintf("a/b/c/file%d", i)
-					bufs[i] = make([]byte, fileSize)
-					for j := 0; j < fileSize; j++ {
-						bufs[i][j] = byte(i*fileSize + j)
-					}
-					err := cb(truncate(files[i], 0))
-					if err != nil {
-						return err
-					}
+				var n int
+				cb(getBenchN(&n))
+				buf := make([]byte, numFiles*fileSize+fileSize)
+				for i := 0; i < numFiles*fileSize; i++ {
+					buf[i] = byte(i)
```

Comment:
This value doesn't really matter, right? Because it'll get truncated. Maybe a comment to indicate that it's just to populate the buffer with non-zero values.

Reply:
The value sort of matters because it gets "truncated" to the full size of the file, leaving all the bytes there, just as a way to force a sync. So I wanted the bytes to all be different, to make sure we didn't trigger any weird dedup stuff (which should be off for journal tests anyway, but better safe than sorry).

I'll add a comment though to be extra clear.

Reply:
Oh I mean that `i` gets truncated to `0-255`, and just loops there over those numbers. But yeah I get your point.